fdbserver/EncryptKeyProxyInterface.h

```diff
@@ -121,20 +122,43 @@ struct EKPGetBaseCipherKeysByIdsReply {
 	}
 };
 
+struct EKPGetBaseCipherKeysRequestInfo {
+	constexpr static FileIdentifier file_identifier = 2180516;
+	EncryptCipherDomainId domainId;
+	EncryptCipherBaseKeyId baseCipherId;
+	EncryptCipherDomainName domainName;
```

Comment:
Is it required to pass the domain name for decryption? Since the encryption header doesn't store domain name, decryption needs to check the domain id -> domain name map. And since this map is available in FDB, can the map lookup logic somehow hide behind encrypt key proxy?

Reply:
> Since the encryption header doesn't store domain name, decryption needs to check the domain id -> domain name map.

So far I think decryption should work without `tenant-name` as the lookup for the cipher-key should be done via cipherKeyId (plus all FDB local caches are indexed by either domainId or cipherKeyId). Tenant-name is useful in mapping ancillary information while refreshing encryption-keys from external KMS, however, KMS point encryption lookup should work with {cipherKeyId, domainId} for all purposes.